recipes-extended/tmux/tmux_3.%.bbappend

```diff
@@ -0,0 +1,15 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+CONFFILES += "\
+    file://tmux.conf \
+"
+
+FILES:${PN}     += "${sysconfdir}/tmux.conf"
+CONFFILES:${PN} += "${sysconfdir}/tmux.conf"
```

Comment:
It doesn't functionally matter in this case, but move packaging variables like this below the `do_install()` (and other task-overrides), per [the styleguide](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/docs/styleguide.md#layout). Keeping to that standard ordering helps make recipes more readable and to avoid obscure errors with ordering that can sometimes occur when you have more complicated recipes that inherit behavior from bbclasses.

Reply:
I didn't look at enough of them. The ones I looked at (the top part of the find all CONFFILES) had them before `do_install()`. If I look further down the list, I do see where these types of commands are after `do_install()`. I will move them since that is actually what I prefer and only moved them to be consistent with what I had found.

Reply:
Moved